.gitignore

```diff
@@ -4,7 +4,6 @@
 Archive
 .DS_Store
 .*.swp
-buildfiles
```

Comment:
This seems wrong. I know you may need to commit a buildfiles for a test case data. But in general we need to have this here.

Reply:
Why? They are always in the target/generated directory and already ignored?

Reply:
True. It was added long before generated was added.